python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.057557,
+          "end_time": "2021-05-18T20:36:45.043130",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:44.985573",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Deploy a model using Prebuilt Docker Images for Inference\n",
+        "\n",
+        "In this tutorial, you will deploy a PyTorch model using the Azure Machine Learning Python SDK. \n",
+        "The model is trained to classify chickens and turkeys by first using a pretrained ResNet18 model that has been trained on the [ImageNet](https://image-net.org/index) dataset."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.2322,
+          "end_time": "2021-05-18T20:36:45.333348",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.101148",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Prerequisites\n",
+        "\n",
+        "If you are using an Azure Machine Learning Notebook VM, you are all set. Otherwise, go through the [Configuration](../../../../configuration.ipynb) notebook to install the Azure Machine Learning Python SDK and create an Azure ML `Workspace`"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.044699,
+          "end_time": "2021-05-18T20:36:45.433973",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.389274",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Check core SDK version number\n",
+        "import azureml.core\n",
+        "\n",
+        "print(\"SDK version:\", azureml.core.VERSION)"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "collapsed": true,
+        "gather": {
+          "logged": 1620920632022
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.068634,
+          "end_time": "2021-05-18T20:36:45.559675",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.491041",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Initialize workspace\n",
```

Comment:
simplify, get to the interesting part of the tutorial sooner (assume the user has read the python-sdk/README.md, has a workspace and compute targets)